save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/ProjectSettingsMenu.kt

```diff
@@ -68,6 +73,21 @@ fun projectSettingsMenu(
     val projectPath = props.project.let { "${it.organization.name}/${it.name}" }
 
     val (wasConfirmationModalShown, setWasConfirmationModalShown) = useState(false)
+    val (isOpenGitWindow, setOpenGitWindow) = useState(false)
+    val (gitDto, setGitDto) = useState(props.gitInitDto)
```

Comment:
Doesn't having a separate copy of `git` object here break data synchronization with parent component? I think the more correct solution (if parent component requires `git`) is to pass in props both `git` and a callback to update its value in parent's state

Reply:
Fixed